apps/crn-server/src/data-providers/contentful/user.data-provider.ts

```diff
@@ -523,6 +575,33 @@ export const parseTeamsCollection = (
     [],
   );
 
+export const parseResearchOutputsCollection = (
+  researchOutputsCollection: ResearchOutputsCollection,
+  userId: string,
+): UserResearchOutput[] =>
+  (researchOutputsCollection?.items || []).reduce(
+    (
+      userResearchOutputs: UserResearchOutput[],
+      researchOutput: ResearchOutputItem | null,
+    ): UserResearchOutput[] => {
+      const isAuthor = researchOutput?.authorsCollection?.items.some(
+        (author) => author?.__typename === 'Users' && author.sys.id === userId,
+      );
+
+      if (isAuthor) {
+        return [
+          ...userResearchOutputs,
+          {
+            id: researchOutput?.sys.id || '',
+          },
```

Comment:
I'm thinking we probably don't need this to be an object with just the id and we can probably just return an array of ids. What do you think?